swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory 
+        self.item = Item()
```

Comment:
be careful passing in data structures like lists and dictionaries as default parameters. No matter how many instances of Vendor are created, they will all share the exact same default list instead of having its own unique list. 

a better option would be:

```suggestion
    def __init__(self, inventory = None):
        if inventory == None:
            self.inventory = []
        else: 
            self.inventory = inventory
        self.item = Item()
```